baton-maven-plugin/src/main/java/org/technologybrewery/baton/BatonMojo.java

```diff
@@ -231,6 +252,6 @@ protected FileSet[] getFileSetsForTarget(MigrationTarget target) {
             localFileSets.add(fileSet);
         }
 
```

Comment:
I: edge case found while unit testing; won't happen in real use, but makes this more predictable when you hack it in tests